### ISSUE TITLE
Whip teammates doesn't save in config (very minor issue)

### ISSUE THREAD

**Issue by ShiroSeby** [tf2cheater2013/Fedoraware#147]
**Fedoraware Version Date: April 5 2022 10:56:21**  

## Config used:
Any config

## Description:  
Whip teammates doesn't save in config, so whenever you save a config with the Whip teammates ticked on it won't save

## Steps to reproduce:  
- [x] I am able to consistently reproduce this error

1. Load fedware
2. Load a config that doesn't have the Whip teammates ticked on
3. Whip teammates will still be active
tl;dr Whip teammates doesn't save on the config file

## Additional context 
tl;dr Whip teammates doesn't save on the config file

- [x] I have included the steps to reproduce to the best I am able to share.

**Comment by xxmyn** [comment]
you could fix this yourself.
